server/features/client_modules.feature

```diff
@@ -6,9 +6,13 @@ Feature: Client modules operations
         Then we get list with 0 items
 
     Scenario: List blogs without needing auth
+    	Given "themes"
+        """
+        [{"name": "forest"}]
+        """
         Given "blogs"
         """
-        [{"title": "testBlog one"}, {"title": "testBlog two"}]
+        [{"title": "testBlog one", "blog_preferences": {"theme": "forest", "language": "fr"}}, {"blog_preferences": {"theme": "forest", "language": "fr"}, "title": "testBlog two"}]
```

Comment:
actually it isn't but we have two blogs, each of them having their own `blog_preferences`